pages/_type/_id/version.vue

```diff
@@ -1,48 +1,8 @@
 <template>
-  <div class="content card">
-    <ConfirmPopup
-      ref="delete_version_popup"
-      title="Are you sure you want to delete this version?"
-      description="This will remove this version forever (like really forever)."
-      :has-to-type="false"
-      proceed-label="Delete version"
-      @proceed="deleteVersion()"
-    />
-    <div class="columns">
-      <nuxt-link
-        v-if="mode === 'version'"
-        class="iconified-button back-button"
-        :to="`/${project.project_type}/${
-          project.slug ? project.slug : project.id
-        }/${
-          $nuxt.context.from
-            ? $nuxt.context.from.name === 'type-id-changelog'
-              ? 'changelog'
-              : 'versions'
-            : 'versions'
-        }`"
-      >
-        <BackIcon aria-hidden="true" />
-        Back to list
-      </nuxt-link>
-    </div>
-    <div v-if="version">
-      <div v-if="mode === 'version'" class="version-header">
-        <h2>{{ version.name }}</h2>
-
-        <div v-if="version.featured" class="featured">
-          <StarIcon aria-hidden="true" />
-          Featured
-        </div>
-        <div
-          v-else-if="featuredVersions.find((x) => x.id === version.id)"
-          class="featured"
-        >
-          <StarIcon aria-hidden="true" />
-          Auto-featured
-        </div>
-      </div>
-      <div v-if="mode === 'edit'" class="header-buttons buttons">
+  <div>
```

Comment:
Is this div with no class/style/whatever needed?

Reply:
I can remove it but then it gives me an error saying the template component needs to have exactly one root component. I remember adding the extra div because I added a card above the original version creation page to hold the validation errors.